.projenrc.js

```diff
@@ -39,6 +39,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
     'aws-cdk-lib@^2.0.0',
     'constructs@^10.0.5',
     'got',
+    'hpagent',
```

Comment:
Please also add `hpagent` in `bundledDeps` at `.projenrc.js`  . Because the `lambda/install.js` is a standalone scripts that is running outside of cdk world.